support-models/src/main/scala/com/gu/support/zuora/api/SubscribeRequest.scala

```diff
@@ -29,33 +29,3 @@ object SubscribeRequest {
 //The subscribe request documented here: https://www.zuora.com/developer/api-references/older-api/operation/Action_POSTsubscribe/
 //fields are upper case to match the expected json structure
 case class SubscribeRequest(subscribes: List[SubscribeItem])
-
-import com.gu.support.encoding.Codec._
-
-object UpdateRedemptionDataRequest {
-  implicit val encoder: Encoder[UpdateRedemptionDataRequest] = deriveEncoder[UpdateRedemptionDataRequest].mapJsonObject(
-    _.renameField("gifteeIdentityId", "GifteeIdentityId__c")
-      .renameField("requestId", "CreatedRequestId__c")
-      .renameField("giftRedemptionDate", "GiftRedemptionDate__c"),
-  )
-}
-
-case class UpdateRedemptionDataRequest(
-    requestId: String,
-    gifteeIdentityId: String,
-    giftRedemptionDate: LocalDate,
-    currentTerm: Int,
-    currentTermPeriodType: PeriodType,
-)
-
-object DistributeRevenueRequest {
```

Comment:
I think we also have a lambda somewhere that distributes revenue, because we found that if we tried to do it immediately, the invoice + revenue schedules had not been created by zuora, causing a support workers failure.

Reply:
Well remembered, I'll do a PR in support-service-lambdas for that.

Reply:
https://github.com/guardian/support-service-lambdas/pull/2654